.code-samples.meilisearch.yaml

```diff
@@ -1188,6 +1188,39 @@ index_settings_tutorial_api_put_setting_1: |-
 index_settings_tutorial_api_task_1: |-
   curl \
     -X GET 'http://localhost:7700/tasks/TASK_UID'
+get_embedders_1: |-
+  curl \
+    -X GET 'http://localhost:7700/indexes/INDEX_NAME/settings/embedders'
+update_embedders_1: |-
+  curl \
+    -X PATCH 'http://localhost:7700/indexes/INDEX_NAME/settings' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "embedders": {
+        "default": {
+          "source": "huggingFace",
```

Comment:
I would not use hugging face example since it does not really work on the cloud. Can we use OpenAI instead?